tests/integration-tests/conftest.py

```diff
@@ -728,6 +733,13 @@ def inject_additional_config_settings(cluster_config, request, region, benchmark
                         compute_resources.pop("Instances")
                         compute_resources["InstanceType"] = instance_type
 
+    # Force addition of ElasticIp as True for Multi Nic instance
+    if request.config.getoption("force_elastic_ip"):
+        if not dict_has_nested_key(config_content, ("HeadNode", "Networking", "ElasticIp")):
+            dict_add_nested_key(config_content, "true", ("HeadNode", "Networking", "ElasticIp"))
+        elif dict_has_nested_key(config_content, ("HeadNode", "Networking", "ElasticIp")):
```

Comment:
what is the mening of this elif case? If I'm not wrong you're doing the same thing in both the cases so you can avoid the if/elif.